backends/platform/sdl/morphos/morphos.cpp

```diff
@@ -52,6 +52,74 @@ bool OSystem_MorphOS::hasFeature(Feature f) {
 	return OSystem_SDL::hasFeature(f);
 }
 
+void OSystem_MorphOS::initBackend() {
+
+	// First time user defaults
+	ConfMan.registerDefault("audio_buffer_size", "2048");
+	ConfMan.registerDefault("fullscreen", false);
+	ConfMan.registerDefault("gfx_mode", "surfacesdl");
+	ConfMan.registerDefault("stretch_mode", "stretch");
+	ConfMan.registerDefault("gui_mode", "antialias");
+	ConfMan.registerDefault("gui_theme", "scummremastered");
+	ConfMan.registerDefault("gui_scale", "125");
```

Comment:
Quite a few of these are either subjective or already the default. Could you limit it to just the changes that are necessary? In particular, could you get rid of the `gfx_mode` override in preparation for PR #6186?

Reply:
ok i keep only some path and audio buffer

Reply:
@ccawley2011 

I have the same in amigaos, do you want me to get rid of them too?

Reply:
> I have the same in amigaos, do you want me to get rid of them too?

If possible, that would be good.

Reply:
np

all of them?

Reply:
The default paths and audio buffer size can stay (as is the case for this PR), the rest can be removed.

Reply:
done...

thank you